doc/introduction.rst

```diff
@@ -0,0 +1,62 @@
+.. _introduction:
+
+============
+Introduction
+============
+
+.. _api_imblearn:
+
+API's of imbalanced-learn samplers
+----------------------------------
+
+The sampler available follows the scikit-learn API using the estimator base
```

Comment:
using the base estimator and adding a sampling functionality throw the ``sample`` method